client/opentracing/src/main/java/org/hawkular/apm/client/opentracing/TraceContext.java

```diff
@@ -125,6 +141,9 @@ public void endProcessingNode() {
             if (sampled) {
                 recorder.report(trace);
             }
+
+            rootNode.addProperty(this.serviceName);
```

Comment:
Should check to make sure application didn't provide the service name on the node already - so don't overwrite if already provided.